tests/test_ci/TestControllers.py

```diff
@@ -760,7 +759,13 @@ def repos(self, *args):
             response = c.post(
                 '/start-ci', environ_overrides=WSGI_ENVIRONMENT,
                 data=json.dumps(data), headers=self.generate_header(data, 'workflow_run'))
-        mock_queue_test.assert_called_once()
+            mock_queue_test.assert_called_once()
+            mock_queue_test.reset_mock()
+            fakedata['workflow_runs'][0]['name'] = Workflow_builds.WINDOWS
+            response = c.post(
+                '/start-ci', environ_overrides=WSGI_ENVIRONMENT,
+                data=json.dumps(data), headers=self.generate_header(data, 'workflow_run'))
```

Comment:
Please separate the test into two tests instead of randomly putting the windows workflow after one test succeeds. Separate tests will help us check for failure in only one without affecting the other

1. test_webhook_workflow_run_completed_successful_linux
2. test_webhook_workflow_run_completed_successful_windows